azure-pipelines.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     displayName: 'Test'
   - script: ./helpers/update-site.sh
     displayName: 'Deploy'
-    condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
+    # condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
```

Comment:
Has the behavior changed? IIRC I had to add `condition` because:
- The pipeline was run if we had different branches and PRs
- It didn't take into account if the previous steps had failed and continued the execution

Reply:
We are just trying to validate to make sure changes are good by making the pipeline run on this branch :) It will be set back to normal before merging.

Reply:
Got it!